cirq/google/calibration/phased_fsim.py

```diff
@@ -130,3 +140,217 @@ def override_by(self, other: 'PhasedFSimCharacterization') -> 'PhasedFSimCharact
             other that are not None). Otherwise the current values are used.
         """
         return other.merge_with(self)
+
+
+# TODO: Add start and end calibration timestamp
+# TODO: Add export to Panda's data frame
+@dataclasses.dataclass(frozen=True)
+class PhasedFSimCalibrationResult:
+    """The PhasedFSimGate characterization result.
+
+    Attributes:
+        parameters: Map from qubit pair to characterization result. For each pair of characterized
+            quibts a and b either only (a, b) or only (b, a) is present.
+        gate: Characterized gate for each qubit pair.
+    """
+
+    # TODO: Add validation that only either (a, b) or (b, a) is present.
```

Comment:
do or open and link issue

Reply:
Moved all the TODOs to a separate doc for now.